config.go

```diff
@@ -144,6 +151,7 @@ func (c *Config) connectContextMaker() (context.Context, func()) {
 }
 
 const defaultMTU = 1200 // bytes
+var defaultCurves = []elliptic.Curve{elliptic.X25519, elliptic.P256, elliptic.P384}
```

Comment:
Our linter doesn't like this one, but I'm not sure what else to do. I can't make it a `const` since Go doesn't accept `const` slices.